CHANGELOG.md

```diff
@@ -1,3 +1,71 @@
+# v0.5.0 (2020-08-13)
+
+Special thanks to first-time contributor @spoonofpower ([#988])!
+
+## Breaking changes
+
+* Remove support for unsigned datastore migrations ([#976])
+
+## OS changes
+
+* Add `aws-ecs-1` variant prototype for running containers in ECS clusters ([#946])
+* Configurable `clusterDomain` kubelet setting via `settings.kubernetes.cluster-domain` ([#988])
+* Make update position within waves consistent ([#993])
+* Fix kubelet configuration for `MaxPods` ([#994])
+* Update `eni-max-pods` with new instance types ([#994])
+* Fix `max_versions` unit test in `updata` ([#998])
+* Add configurable logging drivers for `aws-ecs-1` variant for use with ECS ([#1005])
+* Add IAM task role capabilities support for `aws-ecs-1` variant ([#1007])
```

Comment:
Don't all of these get listed as 'added a new variant for ECS`... #1005, #10007, etc